### PR TITLE
added support for subscription

### PR DESCRIPTION
**Pull request by kelchy**

### CODE REVIEW

**Comment by ronkorving** [comment]
Looks good to me. @AlmirKadric, anything you wanna add?

**Comment by AlmirKadric** [comment]
My only feedabck would be to update https://github.com/Wizcorp/node-iap/blob/master/bin/verify-google.js to also allow the testing of subscriptions.

Otherwise LGTM :+1:

**Comment by ronkorving** [comment]
@kelchy could you add a test please?

**Comment by kelchy** [comment]
can you check?

File: bin/verify-google.js [hunk 1]
```diff
@@ -2,10 +2,10 @@
 
 var fs = require('fs');
 var path = require('path');
-var argv = require('minimist')(process.argv.slice(2), { string: ['productId', 'packageName', 'receipt', 'keyFile'] });
+var argv = require('minimist')(process.argv.slice(2), { string: ['productId', 'packageName', 'receipt', 'keyFile', 'subscription'] });
```
Comment (ronkorving) [inline bin/verify-google.js]: You don't want the string `true`, you want a boolean, right? Minimist has a better way to achieve that.

**Comment by kelchy** [comment]
is this already merged?

**Comment by ronkorving** [comment]
Not yet, sorry for the delay. I wonder how iOS deals with subscription checks and if we can support that. That's not a requirement for this PR to be merged btw.

Also, could you make sure to run `make test`? I doubt this code lints...

**Comment by ronkorving** [comment]
@AlmirKadric we need to integrate a test framework (personally more into `tape` than `mocha` these days, but anyway...)

**Comment by kelchy** [comment]
how can i resolve these conflicts?

**Comment by ronkorving** [comment]
`git pull` the master branch from here into your local repository, fix conflicts, and `git commit -a` to confirm a successful merge. Then push your branch to your `origin` remote again.

**Comment by kelchy** [comment]
done fixing conflict

**Comment by ronkorving** [comment]
Thanks. Is there any way to test this without having a google account set up?

**Comment by kelchy** [comment]
unfortunately, i am not aware if there is any way. we'll just have to watch out for any bug report.

**Comment by ronkorving** [comment]
To what extent have you tested this yourself?

**Comment by kelchy** [comment]
I have deployed this in production for our app.
realistically speaking, we don't have much subscription but there are quite a few so I know it works.

**Comment by ronkorving** [comment]
Thanks. For now, that's good enough for me.

**Pull request merged**